profiles/schema.py

```diff
@@ -0,0 +1,76 @@
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+
+
+class DivisionOfInterestType(DjangoObjectType):
+    class Meta:
+        model = DivisionOfInterest
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+
+    concepts_of_interest = graphene.List(ConceptType)
+    divisions_of_interest = graphene.List(AdministrativeDivisionType)
+
+
+class ProfileInput(graphene.InputObjectType):
+    user_id = graphene.ID()
+    nickname = graphene.String(required=True)
+    image = graphene.String()
+    email = graphene.String()
+    phone = graphene.String()
+    language = graphene.String()
+    contact_method = graphene.String()
+
+
+class CreateProfile(graphene.Mutation):
+    class Arguments:
+        profile = ProfileInput(required=True)
+
+    ok = graphene.Boolean()
+    profile = graphene.Field(ProfileType)
+
+    @login_required
+    def mutate(self, info, **kwargs):
+        profile_data = kwargs.pop("profile")
+        profile = Profile.objects.create(**profile_data)
+        ok = True
+        return CreateProfile(profile=profile, ok=ok)
+
+
+class Query(graphene.ObjectType):
+    profile = graphene.Field(ProfileType, id=graphene.Int())
+    profiles = graphene.List(ProfileType)
+    divisions_of_interest = graphene.List(DivisionOfInterestType)
+
+    @login_required
+    def resolve_profiles(self, info, **kwargs):
+        return Profile.objects.filter(user=info.context.user)
```

Comment:
I think this is what `resolve_profile` should do and it would be enough for now. No need for `resolve_profiles` AFAICT